Back End/MemorizeWords/MemorizeWords/Api/ApplicationVariableApiInitializer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace MemorizeWords.Api
+{
+    public static class ApplicationVariableApiInitializer
+    {
+        public static void Initialize(WebApplication app, IConfiguration configuration) {
+
+            app.MapGet("/api/ApplicationVariable", () =>
```

Comment:
update this 
app.MapGet("/api/ApplicationVariable", () =>
to this
app.MapGet("/api/ApplicationVariable", (IConfiguration configuration) =>